### DISCUSSION TITLE
Insert a hard break without loosing formatting.

### DISCUSSION THREAD

**Discussion by hans-pragt** [ueberdosis/tiptap#1933] [General Questions] [answered]
If I enable the hardbreak extension and extend it to listen for the enter key, starting a paragraph in bold, pressing enter, and further typing will result in:
```
<p><strong>This is a test</strong><br>this is also a test</p>
```
Even though the bolding was never explicitly turned off. How do I prevent this behavior and keep bold enabled across lines?

**Comment by philippkuehn** [comment, marked as answer]
Hey, thanks for that feature request. I added this feature in the latest version. Should be working now. More details here: https://github.com/ueberdosis/tiptap/issues/1934

**Reply by hans-pragt** [reply]
This is great! Thank you!
